cmd/acra-connector/acra-connector.go

```diff
@@ -255,8 +255,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	// if log format was overridden
-	logging.CustomizeLogging(*loggingFormat, ServiceName)
+	logging.Customize().SetFormat(*loggingFormat).SetServiceName(ServiceName).SetOutput(os.Stderr).Complete()
```

Comment:
We don’t need to add `SetOutput(os.Stderr)` here and in other places because that’s what Customize() does by default now.